war_or_peace_runner.rb

```diff
@@ -0,0 +1,79 @@
+class Start
+    attr_reader
+    attr_accessor
```

Comment:
If we aren't defining any `attr_readers` or `attr_accessors` we don't need to list these here.